addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21431,7 +21436,13 @@ msgctxt "#38111"
 msgid "This category contains other settings for the GUI interface"
 msgstr ""
 
-#empty strings from id 38112 to 38189
+#. Description of setting with label #12381 "Switch to visualisation on playback"
+#: system/settings/settings.xml
+msgctxt "#38112"
+msgid "Kodi will automatically go to the visualisation window when audio playback starts"
```

Comment:
I suggest not to hardcode the product name in any UI strings as this prevents clean and simple rebranding.

Reply:
what it actually does is switching to fullscreen on beginning of every new song. not sure if this is what you want to achieve with this change.